docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,229 @@
+
+Every once in a while, a library gets an update that fixes a vulnerability.
+Our security team is on the lookout for critical vulnerabilities and will quickly warn all impacted teams if an update needs to be done.
+
+We also configured [GitHub’s Dependabot Alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts) that will inform teams that a vulnerability has been discovered and which version they should upgrade to to be safe (more on this later).
+
+This approach, while it mostly works, can be very risky.
+If the project hasn’t been touched for quite a while, the number of dependencies to update can be high depending on the vulnerability and the time to update will increase significantly.
+
+### Opportunistic
+
+This rule is simple, when you receive a task on a project, the first step is to upgrade its dependencies.
+Once that is done, you can go ahead with the implementation of your task.
+
+This approach is similar to fixing a bit of tech debt with every business project you take on, it helps to stay ahead in case a vulnerability comes up and keep a baseline for all your projects.
+
+### Update your dependencies regularly
+
+This one sounds easy, every few weeks you check what’s outdated, bump the versions and run your CI.
+If it succeeds, you push the change.
+Maybe there is even a way to automate that?
+
+#### Let's give Renovate a try
+
+One day in November 2019, I discovered that [Renovate](https://www.mend.io/free-developer-tools/renovate/) provides a Docker image that you can run on premise with your own package registries.
+The app creates Pull Requests automatically to inform you of dependencies updates and the CI can then build it automatically so you know if it’s safe to merge or not.
+
+On our first try, we enabled 30 repositories, a cron task was running every hour to create Pull Requests.
+
+We received 700 Pull Requests in the first month, it was a never ending Pull Request whack-a-mole every time we merged one, another replaced it.
+
+![A cat playing Whack-a-mole with a finger](./swissquote_cat_whack_a_mole.gif)
+Me and my team merging Pull Requests.
+
+The awesome thing with Renovate is that it’s very configurable, and this configuration can be shared.
+Very early on we created a shared configuration for our team with some custom policies, here are a few things we decided to do:
+
+- Group PRs for `minor` and `patch` dependencies
+- Internal dependencies could create a PR anytime of the day
+- Third party dependencies could create PRs only during the weekend
+
+This helped a lot to reduce the noise in PRs, the second month we got 400 Pull Requests, and on the third month only 200.
+
+## What we learned from automating dependencies updates
+
+- You need to be confident that your code coverage will warn you in case of updates. At the beginning we missed quite a few breaking updates because the build was green but the application broke as soon as it was deployed.
```

Comment:
```suggestion
- You need to be confident that your code coverage will warn you in case of updates. At the beginning we missed quite a few breaking updates because the build was green but the application broke as soon as it was deployed
```